Rakefile

```diff
@@ -223,3 +223,40 @@ namespace :crowbar do
     end
   end
 end
+
+namespace :test do
+  task dependencies: [] do
+    if ENV["TRAVIS"]
+      system("pip install bashate")
+    else
+      system("sudo pip install bashate")
+    end
+    exit $?.exitstatus
+  end
+
+  desc "Run bashate tests"
+  task bashate: [:dependencies] do
+    script = "scripts/install-chef-suse.sh"
```

Comment:
would be nice to keep it as an extensible for loop like the code before did in case there will be additional scripts to check in the future